manim/config/utils.py

```diff
@@ -56,215 +47,885 @@ def config_file_paths():
     return [library_wide, user_wide, folder_wide]
 
 
-def make_config_parser():
-    """Make a ConfigParser object and load the .cfg files."""
+def make_config_parser(custom_file=None):
+    """Make a ConfigParser object and load the .cfg files.
```

Comment:
Detailed documentation for module-level functions is currently not rendered it seems (the oneline-summary is rendered in the list of functions, but documentation for parameters and notes etc. are not) -- we should take care of this in a separate PR, though.